metrics/metrics.go

```diff
@@ -54,6 +83,13 @@ func Middleware(registry prometheus.Registerer) gin.HandlerFunc {
 		Help:      "A gauge of the number of requests currently being handled.",
 	}, []string{"blocking"})
 
+	outboundCallDuration := prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "http",
+		Name:      "outbound_request_call_seconds",
```

Comment:
This should be namespace `http` and name `client_request_duration_seconds`. We're using this in the connector to measure outbound/upstream HTTP request. It'll be nice to follow the same format.

Reply:
Do we want to use the same metric name for outbound requests to the infra API and outbound requests to provider APIs? Do we ever expect to need to aggregate those? I think we probably want a different metric name for those so that it's a bit easier to consume.

Reply:
Went with `outbound_request_duration_seconds`

Reply:
After a closer look, this probably should be using `namespace=oidc` since it's not using an HTTP client directly